components/shared/constellation/from_script_message.rs

```diff
@@ -214,6 +214,7 @@ pub struct SWManagerSenders {
 
 /// Messages sent to Service Worker Manager thread
 #[derive(Debug, Deserialize, Serialize)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
why this is needed now, this code did not change

Reply:
@Taym95 < Looks like the `enum ServiceWorkerMsg `is too big and `clippy` is complain about it `(the entire enum is at least 472 bytes)`

```
ServiceWorkerMsg
-> ForwardDOMMessage(DOMMessage, ServoUrl),
--> DOMMessage
---> StructuredSerializedData
----> + pub offscreen_canvases: Option<HashMap<OffscreenCanvasId, TransferableOffscreenCanvas>>,
```